web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.tsx

```diff
@@ -85,7 +84,7 @@ export const AnytimeActionDropdown = (props: Props): ReactElement => {
   return (
     <div className={"anytime-action-dropdown-container"}>
       <Heading level={2}>{Config.dashboardAnytimeActionDefaults.defaultHeaderText}</Heading>
-      <HorizontalLine />
+      <hr className="bg-base-lighter margin-top-2 margin-bottom-4" aria-hidden={true} />
```

Comment:
[dust] Any reason you want to replace rather than extend the component?

Reply:
Didn't even think of that but I love that, I'll extend it to use that as a prop instead